frappe/database/database.py

```diff
@@ -772,7 +772,9 @@ def get_single_value(self, doctype, fieldname, cache=True):
 
 		if not df:
 			frappe.throw(
-				_("Invalid field name: {0}").format(frappe.bold(fieldname)), self.InvalidColumnName
+				_("Field {0} does not exist on {1}").format(
+					frappe.bold(fieldname), frappe.bold(doctype), self.InvalidColumnName
```

Comment:
even the error seems wrong? should not be `InvalidColumnName`. Maybe `DoesNotExistError`?

Reply:
Ignoring for now, this feels like another mess. It's used in other places :neutral_face: